Lib/KeychainAccess/Keychain.swift

```diff
@@ -887,12 +887,18 @@ public final class Keychain {
 
     // MARK:
 
-    public class func allKeys(_ itemClass: ItemClass) -> [(String, String)] {
+    public class func allKeys(_ itemClass: ItemClass, accessGroupsCompatible: Bool = true) -> [(String, String)] {
```

Comment:
So I added an optional `accessGroupsCompatible` that defaults to `true` (per Apple documentation Tip)